### PR TITLE
Configuring for pure-python

### PR DESCRIPTION
**Pull request by icemac**

### CODE REVIEW

**Comment by mgedmin** [comment]
Oops, apologies, I introduced some merge conflicts for you with #7.

My GitHub inbox was full of notifications and I didn't notice that you already had a PR for this repo.

File: tox.ini [hunk 1]
```diff
@@ -1,30 +1,62 @@
...
+[testenv:coverage]
+basepython = python3
 deps =
     coverage
+    coverage-python-version
+    zope.testrunner
+commands =
+    coverage run -m zope.testrunner --test-path=src []
+    coverage html
+    coverage report -m --fail-under=23
```
Comment (mgedmin) [inline tox.ini:44]: Note that you need special multiprocess coverage magic to compute the coverage correctly for this and other buildout recipes.  See #7.

Comment (icemac) [reply tox.ini:44]: That means we need a special variant in zopefoundation/meta for buildout recipes.

Comment (icemac) [reply tox.ini:44]: See #8 which was built using such a special variant.

**Comment by icemac** [comment]
Superseded by #9.

**Pull request closed without merging**